### DISCUSSION TITLE
又一款rust编辑器——zed

### DISCUSSION THREAD

**Discussion by kalxd** [just-talks/editor#2] [General]
官网：[https://zed.dev/](https://zed.dev/)

看样子是主打协作。里面集成的特性还是挺多的，都是最近流行的技术。

**Comment by adoyle-h** [comment]
协作功能很鸡肋啊。有多少人是会去结对编程的？
我能想到会结对编程的就两种人，一种是远程工作，一种是在工作中求同事帮助。后者大多数情况都能面对面讨论问题，也用不着远程协作。
如果是排查问题，那自己 git push，对方 git pull 直接看代码就能排查问题了，不需要实时协作。
我觉得结对编程的场景很少，需要实时线上协作的更少。

P.S. nvim 也有结对编程插件 https://github.com/jbyuki/instant.nvim

**Comment by adoyle-h** [comment]
迁移讨论：主版块 -> 编辑器版块
